src/main/java/Parser.java

```diff
@@ -0,0 +1,104 @@
+import java.io.IOException;
+import java.time.LocalDate;
+
+/**
+ * Represents a parser which is used to parse user's input.
+ */
+public class Parser {
+    /**
+     * Takes in user's order and passes order to TaskList and Storage.
+     *
+     * @param order User's input order.
+     * @return Response to the user.
+     * @throws IOException
+     */
+    public static String process(String order) throws IOException {
```

Comment:
This method seems quite long and is doing a lot of things. Maybe it could be separated into smaller methods which handle one functionality each?